### PR TITLE
isolating tests from local git configuration

### PR DESCRIPTION
**Pull request by andreaswittig**
Tests are failing with my local git configuration (e.g. commit hook templates, ...). Therefore I tried to isolate the tests from the local environment.

### CODE REVIEW

File: test/test_helper.bash [hunk 1]
```diff
@@ -11,21 +12,26 @@ setup() {
...
 teardown() {
-  delete_repo
+  delete_repo_home
   export PATH="${INITIAL_PATH}"
 }
 
-delete_repo() {
+delete_repo_home() {
   [ -d $TEST_REPO ] && rm -rf $TEST_REPO || true
+  [ -d $TEMP_HOME ] && rm -rf $TEMP_HOME || true
```
Comment (mtdowling) [inline test/test_helper.bash]: Don't we need to re-export the original HOME directory?

**Comment by mtdowling** [comment]
Thanks! I think we need to re-export the original HOME directory at the end of each test and move this logic to the setup/teardown functions so that if an error occurs, the original HOME directory is still restored properly.

**Comment by andreaswittig** [comment]
@mtdowling Absolutely! Added re-export of HOME directory.

**Comment by mtdowling** [comment]
Thanks!

**Pull request merged**